spec/unit/puppet/provider/ssl_pkey/openssl_spec.rb

```diff
@@ -42,7 +42,7 @@
       it 'creates with given password' do
         resource[:password] = '2x$5{'
         allow(OpenSSL::PKey::RSA).to receive(:new).with(2048).and_return(key)
-        allow(OpenSSL::Cipher).to receive(:new).with('des3')
+        allow(OpenSSL::Cipher).to receive(:new).with('aes-256-cbc')
```

Comment:
Should we change this to `expect`? Right now I think the test would also pass if `OpenSSL::Cipher.new` wasn't called at all.

Reply:
Updated it. But I'm an rspec noob. Let me know if this is what you had in mind.